services/subnet_manager/Dockerfile

```diff
@@ -5,7 +5,7 @@ FROM openjdk:8
 MAINTAINER Zhonghao Lyu <zlyu@futurewei.com>
 
 # Route Manager process
-EXPOSE 8080
+EXPOSE 9000
```

Comment:
900X?

Reply:
I set 9000 here because I make every microservice server.port = 9000 when I deploy them in Docker.